docs/root/version_history/current.rst

```diff
@@ -132,6 +132,7 @@ New Features
 * thrift_proxy: added support for :ref:`mirroring requests <envoy_v3_api_field_extensions.filters.network.thrift_proxy.v3.RouteAction.request_mirror_policies>`.
 * udp: allows updating filter chain in-place through LDS, which is supported by Quic listener. Such listener config will be rejected in other connection-less UDP listener implementations. It can be reverted by ``envoy.reloadable_features.udp_listener_updates_filter_chain_in_place``.
 * udp: disallow L4 filter chain in config which configures connection-less UDP listener. It can be reverted by ``envoy.reloadable_features.udp_listener_updates_filter_chain_in_place``.
+* upstream: support max stream duration for upstream connections.
```

Comment:
this should probably reflink the field

Reply:
I added a reflink and I also fixed it to mention "max connection duration" not "max stream duration"